lua/wikis/marvelrivals/Infobox/League/Custom.lua

```diff
@@ -0,0 +1,90 @@
+---
+-- @Liquipedia
+-- wiki=marvelrivals
+-- page=Module:Infobox/League/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local Link = require('Module:Widget/Basic/Link')
+local String = require('Module:StringUtils')
+
+local Injector = Lua.import('Module:Widget/Injector')
+local League = Lua.import('Module:Infobox/League')
+
+local Widgets = require('Module:Widget/All')
+local Cell = Widgets.Cell
+
+---@class MarvelrivalsLeagueInfobox: InfoboxLeague
+local CustomLeague = Class.new(League)
+local CustomInjector = Class.new(Injector)
+
+local DEFAULT_PLATFORM = 'PC'
+local PLATFORM_ALIAS = {
+	console = 'Console',
+	pc = 'PC',
+}
+
+---@param frame Frame
+---@return Html
+function CustomLeague.run(frame)
+	local league = CustomLeague(frame)
+	league:setWidgetInjector(CustomInjector(league))
+
+	return league:createInfobox()
+end
+
+---@param id string
+---@param widgets Widget[]
+---@return Widget[]
+function CustomInjector:parse(id, widgets)
+	local caller = self.caller
+	local args = caller.args
+
+	if id == 'custom' then
```

Comment:
shouldn't it rather go into `gamesettings`?

Reply:
im not familiar with ``gamesettings``. Marvel Rivals doesnt have that as a module (which is what im assuming that is supposed to be)

Reply:
it is a section in infobox league just like `custom`

Reply:
ah, my understanding is that the gamesettings field was only for what version the game/server the event was played on. Is that the preferred field to use instead of a custom field?